controller/konnect/ops/server_url.go

```diff
@@ -18,6 +32,16 @@ func NewServerURL(u string) ServerURL {
 	return ServerURL(u)
 }
 
+// String returns the string representation of the ServerURL.
 func (s ServerURL) String() string {
 	return string(s)
 }
+
```

Comment:
Do you mind adding an example of the replacement performed by this function? Would help to get the context quicker.

Reply:
f5c812d0749c70dfa547fc5dfa09fdd8a1025ed0

PTAL